src/transformers/benchmark/benchmark_tf.py

```diff
@@ -60,9 +60,10 @@ def run_in_graph_mode(*args, **kwargs):
             return func(*args, **kwargs)
 
         if do_eager_mode is True:
-            assert (
-                use_xla is False
-            ), "Cannot run model in XLA, if `args.eager_mode` is set to `True`. Please set `args.eager_mode=False`."
+            if use_xla is True:
```

Comment:
```suggestion
            if not (use_xla is False):
```
Not sure of the full context here, but maybe `use_xla` can be `None`.

Reply:
I'm not sure how strict you are with typehints, but it indicates a bool. Is that sufficient to consider it a bool or should we still consider `None` as an option? 

I can just change it to `if use_xla:` if we can consider it a bool, making things neater.

Reply:
This also goes for some of the changes below (`if is_a` and `if not is_a`).

Reply:
Type-hints are not 100% accurate, and the test explicitly testing `is True` instead of just `if use_xla`. Maybe for wrong reasons but since I wasn't the person who wrote it, let's just take the strict negation of it to be sure :-)

Reply:
Fair enough, let me adjust :)